guide/payments/receive.md

```diff
@@ -0,0 +1,215 @@
+   retina = "/assets/images/guide/payments/receive/enter-details-from@2x.png"
+   alt-text = "Screen asking the receiver to enter the name of the sender"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!--
+Technical nitpick;
+Addresses are not exactly "generated"
+since they are deterministic technically
+what happens is an unused address just gets chosen
+-->
+
+<!-- 
+Update interopability page
+Since there are many Bitcoin applications for people to choose from, its likely that the sender and receiver are not using the same one and they don't have the abilitiy to exchange payment details within the application itself. 
+
+The exchange of payment details then most commonly happens outside of the wallet application. This 
+
+Since bitcoin is a open system and has many payment applications built ontop of it, there is a [standard format]({{ '/guide/foundations/wallet-interoperability/#payment-links' | relative_url }}) of "payment links" that most bitcoin applications use to share payment details.
+-->
+
+## Sharing the payment request
+Whether it’s just an address or a payment link, the two methods of sharing the payment details are text or scannable QR Codes.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share.png"
+   retina = "/assets/images/guide/payments/receive/share@2x.png"
+   alt-text = "Screen showing the invoice with a share button"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share-sharesheet.png"
+   retina = "/assets/images/guide/payments/receive/share-sharesheet@2x.png"
+   alt-text = "Screen showing the invoice overlayed with a share sheet"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+**Text**
+
+Sharing payment details as text is as straightforward as copying and pasting. It is also typical to provide the receiver with a button to conveniently share in other applications, with “share sheets,” which most mobile operating systems offer.
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
```

Comment:
QR codes aren't just used when devices are close together.  They can be shared only and sometimes sent as an image.

Reply:
hmm what do you mean by shared only, like printed or something? Saw a video of some [drones making a QR](https://www.youtube.com/watch?v=G9K9OOAcUWQ) in the sky. In this case it isn't a device with a wallet presenting the QR.

As an image though, for it to be scanned by the camera it still needs to be close to the other device that is presenting the QR. If the QR is on the same device that has to read it, then another form of input data would be more suitable, especially in mobile contexts (text).

Reply:
Help me understand the argument a bit more though, cause not sure I've gotten it, and I'd like to get the QR stuff correct since its so critical in bitcoin world.

Reply:
Meant to say shared online* sorry, which would encompass being sent as an image. Here is an example of this being used:

Receiver generates new address / associated QR. Receiver and sender are not close by so the sender scanning the receivers QR off their device isn't possible. Receiver emails sender the address / QR. Sender scans the QR they received via email (most likely opening the email on their desktop and scanning with their mobile wallet.)

Some wallets, Bluewallet being an example, can actually send to an address through a QR image saved locally in your photos. So the receiver would send the QR as an image, you click send in blue wallet, then click the 'image' icon in bottom left, and click on the image of the QR. On mobile bitcoin payment URI that openly directly in your wallet are probably a better UX but this alternative option still exists.